plugins/currency/nanopool/nanopool_

```diff
@@ -0,0 +1,290 @@
+        print("wshares.min 0")
+        print("wshares.draw LINE1")
+
+
+def write_data_worker(data):
+    worker_data = sorted(data["data"]["workers"], key=lambda element: element["id"])
+
+    print("multigraph worker_hashrate_{0}".format(account_address))
+
+    for i, val in enumerate(worker_data):
+        worker_name = "_".join(val["id"].split())
+        print("worker_{0}_hashrate.value {1}".format(worker_name, str(val["hashrate"])))
+
+    for val in worker_data:
+        print("")
+        worker_name = "_".join(val["id"].split())
+        print("multigraph worker_hashrate_{0}.worker_{1}".format(account_address, worker_name))
+        print("whashrate.value {0}".format(str(val["hashrate"])))
+
+    print("")
+    print("multigraph worker_shares_{0}".format(account_address))
+    for i, val in enumerate(worker_data):
+        worker_name = "_".join(val["id"].split())
+        print("worker_{0}_shares.value {1}".format(worker_name, str(val["rating"])))
+
+    for val in worker_data:
+        worker_name = "_".join(val["id"].split())
+        print("")
+        print("multigraph worker_shares_{0}.worker_{1}".format(account_address, worker_name))
+        print("wshares.value {0} ".format(str(val["rating"])))
+      
+
+
+def write_config():
+    if graph_type not in GRAPH_TYPES.keys():
+        print("Unknown graph type '{0}'".format(graph_type), file=sys.stderr)
+        exit(1)
+    if graph_type == "worker":
+        write_config_worker()
+        return
+    params = GRAPH_TYPES[graph_type]
+    for item in params:
+        print("graph_title {0}".format(item["title"]))
+        print("graph_category other")
+        if "info" in item:
+            print("graph_info {0}".format(item["info"]))
+        if "scale" in item:
+            print("graph_scale {0}".format(item["scale"]))
+        if "args" in item:
+            print("graph_args {0}".format(item["args"]))
+        for field in item["fields"]:
+            print("{0}.label {1}".format(field, field))
+            print("{0}.type {1}".format(field, item["type"]))
+
+def write_suggest():
+    for item in GRAPH_TYPES.keys():
+        print(item)
+
+def write_data():
+    data = request_data()
+
+    if graph_type == "balance":
+        print("ETH.value {0}".format(data['data']['balance']))
+    elif graph_type == "hashrate":
+        print("hashrate.value {0}".format(data["data"]["hashrate"]))
+    elif graph_type == "avghashrate":
+        print("avg_hashrate.value {0}".format(data["data"]["avgHashrate"]["h1"]))
+    elif graph_type == "worker":
+        write_data_worker(data)
+    else:
+        pass
+
+
+if __name__ == "__main__":
+
+    program = sys.argv[0]
+    try:
+        graph_type, account_address = program.split("_")[1:]
+    except ValueError:
+        print("Please configure the plugin as described in the configuration section.")
+        exit(1)
+
+    GRAPH_TYPES = define_graph_types()
+    if len(sys.argv) > 1 and sys.argv[1] == "autoconf":
```

Comment:
Please take a look at the [autoconf documentation](http://munin-monitoring.org/wiki/PluginConcise#autoconf) for some more details.

Since this plugin does not require additional dependencies (e..g non-core python modules), you can simply get rid of `autoconf`, I guess.